packages/node/src/indexer/apiPromise.connection.ts

```diff
@@ -123,6 +124,16 @@ export class ApiPromiseConnection
     await this.unsafeApi.disconnect();
   }
 
+  async updateChainTypes(chainTypes: RegisteredTypes): Promise<void> {
+    // Typeof Decorate<'promise' | 'rxjs'>, but we need to access this private method
+    const currentApiOptions = (this.unsafeApi as any)._options as ApiOptions;
+    const apiOption = {
+      ...currentApiOptions,
+      ...chainTypes,
+    };
```

Comment:
```suggestion
    const apiOption: ApiOptions = {
      ...currentApiOptions,
      chainTypes,
    };
```

Reply:
should be same line as https://github.com/subquery/subql/pull/2505/files#diff-c9f4c413e59e6f6047e10cd5abdc7e5aef91fd9d24a84732459c7d6acf395631R84